BootloaderCorePkg/Stage1B/Stage1B.c

```diff
@@ -278,6 +279,12 @@ SecStartup2 (
 
   DEBUG ((DEBUG_INFO, "\n============= Intel Slim Bootloader STAGE1B =============\n"));
 
+  StageBase = PCD_GET32_WITH_ADJUST (PcdStage1BLoadBase);
```

Comment:
Can we pass this info directly from Stage1A using STAGE1A_HOB ?  I think it might be more generic.

Reply:
Okay, let me add Stage1BBase in STAGE1A_HOB.